rrmngmnt/network.py

```diff
@@ -559,8 +559,8 @@ def send_icmp(self, dst, count="5", size="1500", extra_args=None):
             bool: True/false
         """
         cmd = ["ping", dst, "-c", count, "-s", size]
```

Comment:
You need to remove the size from here.

Reply:
Right! I forgot it (0_0)... will be removed.